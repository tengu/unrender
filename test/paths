/data/band/black-sabbath.json
/data/band/judas-priest.json
/data/band/deep-purple.json
/data/band/iron-maiden.json
/data/band/motörhead.json
/www/favicon.ico
/www/metal.css
/www/html/black-sabbath.html
/www/html/judas-priest.html
/www/html/deep-purple.html
/www/html/iron-maiden.html
/www/html/motörhead.html
