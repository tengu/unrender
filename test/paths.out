/www/favicon.ico
/www/html/*
/www/metal.css
/data/band/*
